readme.md

```diff
@@ -374,6 +384,16 @@
 | ------: | -------: | -----: |
 |   2 - 4 |       12 | 60-90m |
 
+### [Mastermind](https://boardgamegeek.com/boardgame/2392/mastermind)
+
+> Guess the color of hidden pegs. A deduction game where each player takes turn making a limited number of guesses, using logic to deduce what pegs the opponent has hidden.
+
+![Mastermind image](https://target.scene7.com/is/image/Target/GUEST_ed6783bd-a3da-470d-93c5-3b73fc716bfd?wid=488&hei=488&fmt=pjpeg)
```

Comment:
The image is overly large, mostly due to a large amount of padding white space. There's an appropriately sized image from wikimedia that should suffice, even if it's a slightly older version of the board game.

```suggestion
![Mastermind image](https://upload.wikimedia.org/wikipedia/commons/thumb/2/2d/Mastermind.jpg/226px-Mastermind.jpg)
```